stash/src/controller/FaucetController.ts

```diff
@@ -0,0 +1,15 @@
+import * as faucetService from '../service/FaucetService.js'
+import * as errorHandler from '../error/Handler.js'
+import { Request, Response } from 'express'
+import { pathFaucetSchema } from '../schema/FaucetSchema.js'
+
+export const captcha = async (req: Request, res: Response) => {
```

Comment:
There is missing swagger autogeneration documentation.

Reply:
@majusko  endpoint documentation is added in [swagger-output.json](https://github.com/mangata-finance/eigen-layer-monorepo/blob/a6348dc577307247e8ca06eec7ca9fea8cb5cdfd/stash/swagger-output.json#L860) do we need something other than that?

Reply:
@olgakonsta that is autogenerated file, please check other controller methods and follow the documentation pattern based on which the autogeneration works

Reply:
done